packages/react/src/components/DataTable/TableSelectRow.js

```diff
@@ -22,13 +22,16 @@ const TableSelectRow = ({
   disabled,
   radio,
   className,
+  ...rest
 }) => {
   const selectionInputProps = {
     id,
     name,
     onClick: onSelect,
     checked,
     disabled,
+    'aria-expanded': checked,
```

Comment:
Just wondering if this should be `aria-checked`, rather than `aria-expanded` since this seems to be on the checkbox in `DataTable with Selection`? 

Looking into it further, I see that this is also used to convey the fact that the data table toolbar is now "expanded". Seems like this could be confusing, since the row itself isn't expanding, it's the table toolbar that is appearing. 

Let me know if I am missing something 🤔

Reply:
@tw15egan Thank you for asking - I can see that `aria-expanded` for a checkbox may be confusing. This approach does below trick discussed in #5242 wrt how to tell screen readers that the checkbox expands/collapses batch action bar:

From @asudoh:

>> Does ARIA have a notion of "contextual action bar", where an action bar appears only in certain context?

From @carmacleod

> ARIA can be used to indicate that an input like a button or checkbox controls the visibility of a container element. See the spec for aria-expanded. So if each checkbox has aria-expanded="true/false" and uses aria-controls to point to the id of the section element, then that should be helpful.

Does this make sense? Please don't hesitate to ping me if you have any questions on this. Thanks!

Reply:
Makes sense, thanks for the clarification 👍

Reply:
Is there a link where I can test this? Preferably one where there's more than one row? Neither of the above preview links seem to go to a table component page. I just want to make sure the aria-expanded markup isn't confusing with multiple checkboxes, because even though it follows the spec, it is a bit unorthodox.

Does the toolbar show when any one checkbox is checked, and hide when all checkboxes are unchecked? If so, the logic for aria-expanded might be a bit tricky.

Reply:
@carmacleod should be able to view it here: https://deploy-preview-5631--carbon-components-react.netlify.com/?path=/story/datatable--with-selection

Reply:
Thanks, @tw15egan ! Looking at https://deploy-preview-5631--carbon-components-react.netlify.com/iframe.html?id=datatable--with-batch-actions